scriptmodules/libretrocores/lr-flycast.sh

```diff
@@ -10,61 +10,51 @@
 #
 
 rp_module_id="lr-flycast"
-rp_module_desc="Dreamcast emulator - Reicast port for libretro"
-rp_module_help="Previously named lr-reicast then lr-beetle-dc\n\nDreamcast ROM Extensions: .cdi .gdi .chd .m3u, Naomi/Atomiswave ROM Extension: .zip\n\nCopy your Dreamcast/Naomi roms to $romdir/dreamcast\n\nCopy the required Dreamcast BIOS files dc_boot.bin and dc_flash.bin to $biosdir/dc\n\nCopy the required Naomi/Atomiswave BIOS files naomi.zip and awbios.zip to $biosdir/dc"
-rp_module_licence="GPL2 https://raw.githubusercontent.com/libretro/flycast/master/LICENSE"
-rp_module_repo="git https://github.com/libretro/flycast.git master"
+rp_module_desc="Multi-platform Sega Dreamcast, Naomi and Atomiswave emulator derived from Reicast"
```

Comment:
Minor, but do we need to mention multi platform ?

Reply:
I've updated the desc from upstream. Maybe we can change it to 'Open Source ...' instead ?